src/platform/plugins/shared/data_view_management/public/components/edit_index_pattern/source_filters_table/components/add_filter/add_filter.test.tsx

```diff
@@ -8,35 +8,98 @@
  */
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { screen, render, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
 
 import { AddFilter } from './add_filter';
 
+const mockMakeRegExTest = jest.fn(() => true);
+
+jest.mock('@kbn/kibana-utils-plugin/common/field_wildcard', () => {
+  const originalModule = jest.requireActual('@kbn/kibana-utils-plugin/common/field_wildcard');
+  return {
+    ...originalModule,
+    makeRegEx: () => {
+      return {
+        test: mockMakeRegExTest,
+      } as unknown as RegExp;
+    },
+  };
+});
```

Comment:
i think you can simplify all of this by just mocking the whole module since you are not using anything else from it:

```suggestion
import { makeRegEx } from '@kbn/kibana-utils-plugin/common';

jest.mock('@kbn/kibana-utils-plugin/common')
const mockMakeRegExTest = jest.mocked(makeRegEx)
```

Reply:
then it started to fail for me ... so I guess this needs more changes?